src/blocks/discord/mslash/options.js

```diff
@@ -140,8 +140,9 @@ Blockly.Blocks[blockName] = {
 
 
 Blockly.JavaScript[blockName] = function(block) {
-    const string = Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC);
     const searchType = block.getFieldValue("SEARCH");
+    let string = Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC);
+    if (searchType=="String") string=string.toLowerCase();
```

Comment:
- Most efficient & keeping `string` as [`const`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const), but a bit more difficult to read
```suggestion
    const string = searchType == "String" ? Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC).toLowerCase() : Blockly.JavaScript.valueToCode(block, "BOOLEAN", Blockly.JavaScript.ORDER_ATOMIC);
```

Reply:
Does `string` requires to be a `let`?

Reply:
i guess not...

Reply:
- [Comment Reply](https://github.com/scratch-for-discord/Web-Application_Frontend/pull/105/files#r1013346278) from @frostzzone:
  > i guess not...

Then it's better to accept this suggestion